.golangci.yaml

```diff
@@ -5,8 +5,6 @@ run:
 linters:
   enable:
   - asciicheck
-  - deadcode
```

Comment:
these got removed by the newer linter version?

Reply:
I pointed this out in the notes that `deadcode` is now `unused`